### PR TITLE
Update language.js

### PR DESCRIPTION
**Pull request by martijnmcb**
Added first part of Dutch translation till line 2267

### CODE REVIEW

File: lib/language.js [hunk 1]
```diff
@@ -214,7 +225,8 @@ function init() {
       ,dk: 'Onsdag'
       ,fi: 'Keskiviikko'
       ,nb: 'Onsdag'
-      ,he: 'רביעי'
+      ,he: 'רביעי
+      ,nl: 'Woensdag''
       }
     ,'Thursday' : {
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2496909485/issues/source?bid=2452374&fileBranchId=3269876#l231)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2496909485/issues/source?bid=2452374&fileBranchId=3269876#l231)

**Comment by jasoncalabrese** [comment]
@martijnmcb, sorry, but these PRs need to go target the `dev` branch instead of `master`, also to help coordinate translation you may want to join https://www.facebook.com/groups/NightscoutTranslators/ if you haven't already

**Comment by martijnmcb** [comment]
@jasoncalabrese Hi Jason, Can i chance this somewhere? Have a problem understanding the logic of GitHub.

**Comment by jasoncalabrese** [comment]
You can't change the target after creating the pull request, so this 1 will need to be closed and a new 1 opened.  Hopefully there aren't any merge conflicts, but the `language.js` file is a bit of a hot spot with so many edits.

**Pull request closed without merging**